health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -0,0 +1,682 @@
+                                }}
+                            />
+                        </div>
+                        <div style={{ height: "2rem" }}></div>
+                        <div>
+                            {boundaryData.map((item, index) => (
+                                <div key={`boundary-${index}`}>
+                                    <div style={{ fontWeight: "600", fontSize: "1.2rem" }}>
+                                        {/* {item?.boundaryType} */}
+                                        {`${t((defaultHierarchyType + "_" + item?.boundaryType).toUpperCase())}`}
+                                    </div>
+                                    <div style={{ height: "1rem" }}></div>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)